crates/cli-tools/Cargo.toml

```
@@ -22,6 +22,7 @@ humantime = { version = "2.1.0", default-features = false, optional = true }
 indicatif = { version = "0.17.8", default-features = false, optional = true }
 log = { version = "0.4.21", default-features = false }
 rusb = { version = "0.9.4", default-features = false, optional = true }
+semver = "1.0.23"
```

I guess the changelog script checks that we always use `default-features = false`. (Add any feature you need explicitly.)
```suggestion
semver = { version = "1.0.23", default-features = false }
```